docs/01-app/05-api-reference/05-config/01-next-config-js/rewrites.mdx

```diff
@@ -158,6 +158,7 @@ module.exports = {
 }
 ```
 
+> **Good to know**: The useSearchParams hook won't work in this case because the URL doesn't change. To access query parameters, you should use the searchParams argument provided to the page. For more details about page arguments, see the [page documentation](/docs/app/api-reference/file-conventions/page).
```

Comment:
```suggestion
> **Good to know**: The `useSearchParams` hook won't work in rewrites because the URL doesn't change. To access query parameters, you should use the `searchParams` prop passed to the page. See the [`page` documentation](/docs/app/api-reference/file-conventions/page) for more information.
```

Reply:
@delbaoliveira Done !! Thanks !! English is not my mother language all suggest are welcome